libs/tup-components/src/tickets/TicketModal.tsx

```diff
@@ -0,0 +1,379 @@
+              [el.value]
+            )}
+          >
+            Download
+          </Button>
+        ),
+      },
+    ],
+    [ticketId, baseUrl, jwt]
+  );
+
+  const { getTableProps, getTableBodyProps, rows, prepareRow, headerGroups } =
+    useTable({
+      columns,
+      data: json,
+    });
+
+  return (
+    <div className="o-fixed-header-table">
+      <table {...getTableProps()}>
+        <thead>
+          {headerGroups.map((headerGroup) => (
+            <tr {...headerGroup.getHeaderGroupProps()}>
+              {headerGroup.headers.map((column) => (
+                <th key={column.id}>{column.render('Header')}</th>
+              ))}
+            </tr>
+          ))}
+        </thead>
+        <tbody {...getTableBodyProps()}>
+          {rows.length ? (
+            rows.map((row, idx) => {
+              prepareRow(row);
+              return (
+                <tr {...row.getRowProps()}>
+                  {row.cells.map((cell) => (
+                    <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
+                  ))}
+                </tr>
+              );
+            })
+          ) : (
+            <tr>
+              <td colSpan={5}>No attachments to display.</td>
+            </tr>
+          )}
+        </tbody>
+      </table>
+    </div>
+  );
+};
+Attachments.propTypes = {
+  attachments: PropTypes.arrayOf(PropTypes.array.isRequired).isRequired,
+  ticketId: PropTypes.string.isRequired,
+};
+
+const TicketHistoryReply: React.FC<{ ticketId: string }> = ({ ticketId }) => {
+  const mutation = useTicketReply(ticketId);
+  const { mutate, isLoading, isError } = mutation;
+
+  const defaultValues = useMemo(
+    () => ({
+      text: '',
+      files: [],
+    }),
+    []
+  );
+
+  return (
+    <Formik
+      enableReinitialize
+      initialValues={defaultValues}
+      validationSchema={formSchema}
+      onSubmit={(values, { resetForm }) => {
+        const formData = new FormData();
+        formData.append('text', values['text']);
+        if (values.files) {
+          values.files.forEach((file) => formData.append('files', file));
+        }
+        mutate(formData, {
+          onSuccess: () => resetForm(),
+        });
+      }}
+    >
+      {({ isSubmitting, isValid }) => {
+        return (
+          <Form className="ticket-reply-form">
+            <FormikInput
+              name="text"
+              label="Reply"
+              type="textarea"
+              className="ticket-reply-text-area"
+              description=""
+              required
+            />
+            <FileInputDropZoneFormField
+              id="files"
+              isSubmitted={isSubmitting}
+              description="Error reports and screenshots can be helpful for diagnostics"
+              maxSizeMessage="Max File Size: 3MB"
+              maxSize={3145728}
+            />
+            <FormGroup className="ticket-reply-submission">
```

Comment:
**Forenotice.** When converting field wrappers from form styles to Core-Styles form pattern, there is an opportunity to better than I did with `FormGroup`.

<details>

I did bad not good. I should have created a TACC-specific FormGroup. [But instead I replaced it with manual markup.](https://github.com/TACC/tup-ui/pull/70/files#diff-efebf9f8f5c426130482dfc6b681a3921cb64c1b8aeaf24997d0ee72e44a53f1L21-L28) I am still learning Typescript.

</details>